app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/viewmodel/PledgedProjectsOverviewViewModel.kt

```diff
@@ -179,6 +180,14 @@ class PledgedProjectsOverviewViewModel(environment: Environment) : ViewModel() {
         )
     }
 
+    private fun showHeadsUpSnackbar(messageId: Int) {
+        snackbarMessage.invoke(messageId, KSSnackbarTypes.KS_HEADS_UP.name)
+    }
+
+    private fun showErrorSnackbar(messageId: Int) {
+        snackbarMessage.invoke(messageId, KSSnackbarTypes.KS_ERROR.name)
+    }
```

Comment:
Depending on if its an error or normal message you can call one of these methods to display it